yogstation/code/modules/admin/verbs/fix_air.dm

```diff
@@ -51,8 +52,3 @@
 			if(!istype(T, /turf/open/space) && T.is_openturf && !T.blocks_air)
 				T.air?.parse_gas_string(T.initial_gas_mix)
 				T.update_visuals()
```

Comment:
```suggestion
				T.update_visuals()
			CHECK_TICK

	if(atmos_enabled)
		message_admins("Re-enabling atmospherics, air on zlevel fixed")
		SSair.can_fire = atmos_enabled
```

Reply:
No this is hacky and in retrospect I should have gutted this out of adam's code